code/modules/admin/verbs/debug.dm

```diff
@@ -666,17 +666,19 @@ GLOBAL_PROTECT(AdminProcCallSpamPrevention)
 	message_admins("<span class='notice'>[key_name_admin(usr)] changed the equipment of [key_name_admin(M)] to [dresscode].</span>", 1)
 
 /client/proc/robust_dress_shop()
-	var/list/outfits = list(
+	var/list/special_outfits = list(
 		"Naked",
 		"As Job...",
 		"Custom..."
 	)
 
+	var/list/outfits = list()
```

Comment:
I feel like this list should be made static. So that it is not reconstructed and sorted each time an admin uses this proc.
This is a bonus thing though since it will add a lot more to the PR

Reply:
I don't think it's worth it.
It's used like a dozen times at most in any given round. It's not going to give any real performance benefits.

My dislike for statics outweighs the potential gains here.